vault/backup-job/base/trigger/clusterrolebinding.yaml

```diff
@@ -9,4 +9,4 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: backup-job
-  namespace: vault
+  namespace: default
```

Comment:
This is intentional; when using the namespace `default`, kustomize will update the value when there is a `namespace:` set in `kustomization.yaml`.